toolkit/tools/imagecustomizerapi/fileconfig.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
In a lot of ways yaml is better than json for this, but I'm leery of splitting our configs like this. (I'm not opposed to looking at moving to yaml for the other configs in the future, maybe with a translator?).

I'm just worried that we will force outselves to have no code-reuse here (maybe a good thing?)

Reply:
Yeah, the split was intentional, regardless of JSON vs. YAML. Image creation and image customization are different procedures. So, there is a good chance they might need to diverge in the future.